app/app/src/main/java/com/example/clubwat/views/ClubUserManagementView.kt

```diff
@@ -103,7 +103,7 @@ fun ClubUserManagementView(
                                                 clubId
                                             )
                                         }) {
-                                            Text(color = LightOrange, text = "Approve")
+                                            Text(color = LightOrange, text = "Paid")
```

Comment:
you will probably ask about this - its because our FP worded it as we're verifying if someones paid the membership fee so i want to clarify that's what the purpose of this button is for that FP.